src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -81,7 +73,31 @@ protected void runTool(final JavaSparkContext ctx) {
             }
 
         });
-        assembledBreakpoints.saveAsTextFile(output);
+        allContigAlignments.saveAsTextFile(output);
+    }
+
+    /**
+     * Loads an RDD of {@link ContigsCollection} objects keyed by assembly ID from disk. The input file
+     * should be the output of as RunSGAViaProcessBuilderOnSpark.
+     */
+    static JavaPairRDD<String, ContigsCollection> loadContigsCollectionKeyedByAssemblyId(final JavaSparkContext ctx, final String inputPath) {
+        final JavaRDD<String> inputAssemblies = ctx.textFile(inputPath).cache();
+
+        final JavaPairRDD<String, String> contigCollectionByBreakpointId =
+                inputAssemblies
+                        .flatMapToPair(RunSGAViaProcessBuilderOnSpark::splitAssemblyLine);
```

Comment:
It feels very strange to me to be calling static methods from one tool in a different tool.  (Examples: the method we're in: loadContigsCollectionKeyedByAssemblyId, or like this call to splitAssemblyLine in RunSGAViaProcessBuilderOnSpark.
It's probably just an atavism from days of C and C++ where if you tried that you'd end up with two main methods and linking would fail.
Also, I know that the organization of our jar is "a gigantic can of worms", where we have a driver program that can call any tool, and therefore all code with all its complex dependencies must always be available for loading.
But tangling the dependency tree in this way can't be good.
There.  I've had my say.  Feel free to ignore this comment.